fuseimg/datasets/mnist.py

```diff
@@ -45,16 +42,18 @@ def dataset(cache_dir: Optional[str] = None, train: bool = True, sample_ids: Seq
         """
 
         transform = transforms.Compose([transforms.ToTensor(), transforms.Normalize((0.1307,), (0.3081,))])
+
         # Create dataset
-        torch_train_dataset = torchvision.datasets.MNIST(
-            cache_dir, download=cache_dir is not None, train=train, transform=transform
-        )
+        torch_dataset = datasets.MNIST(cache_dir, download=cache_dir is not None, train=train, transform=transform)
+
         # wrapping torch dataset
-        if sample_ids is not None:
-            torch_train_dataset = Subset(torch_train_dataset, sample_ids)
-        train_str = "train" if train else "test"
-        train_dataset = DatasetWrapSeqToDict(
-            name=f"mnist-{train_str}", dataset=torch_train_dataset, sample_keys=("data.image", "data.label")
+        name_str = "train" if train else "test"
```

Comment:
@alex-golts , would it break the one-click cross-validation example?
Do we have a test for it?